PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingViewController.swift

```diff
@@ -175,6 +182,21 @@ class PetListingViewController: UIViewController
         self.present(alertController, animated: true, completion: nil)
     }
     
+    func presentFilterViewController()
+    {
+        let filterVC = storyboard?.instantiateViewController(withIdentifier: "filterVC") as? FilterViewController
```

Comment:
Unwrapping the optional right up here (either `if let` or with a `guard`) would prevent needing to force unwrap `filterVC` down on line 193.